src/main/java/fr/insee/pogues/exception/DeReferencingException.java

```diff
@@ -0,0 +1,12 @@
+package fr.insee.pogues.exception;
+
+/**
+ * Exception thrown if an error occurs during questionnaire de-referencing (composition feature).
+ */
+public class DeReferencingException extends Exception {
```

Comment:
Not sure about this one, could it be better if extending RuntimeException ?

Reply:
I tend to prefer checked expression in java, so that upper level classes have to handle the exception explicitly. 

Based on what I've read, it seems that there is no established convention between the two choices:
- https://stackoverflow.com/a/54454791/13425151
- https://stackoverflow.com/a/36663481/13425151
- https://stackoverflow.com/a/2190177/13425151